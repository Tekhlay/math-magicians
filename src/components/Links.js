/* eslint-disable import/no-extraneous-dependencies */
import React from 'react';
import { Outlet, Link } from 'react-router-dom';
import '../style.css';

const Links = () => (
  <nav>
    <div className="header">
      <h1>Math Magicians</h1>
      <section className="nav-link">
        <Link to="/" className="nav-item"> Home</Link>
        <h2>|</h2>
        <Link to="/calculator" className="nav-item">Calculator</Link>
        <h2>|</h2>
        <Link to="/Quote" className="nav-item">Quote</Link>
      </section>
    </div>
    <Outlet />
  </nav>
);

export default Links;
