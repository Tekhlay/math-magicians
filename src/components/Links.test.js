import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Links from './Links';

const LinkMock = () => {
  <BrowserRouter>
    <Links />
  </BrowserRouter>;
};
describe('Test Link components', () => {
  it('render correctly', () => {
    const tree = renderer.create(<LinkMock />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
