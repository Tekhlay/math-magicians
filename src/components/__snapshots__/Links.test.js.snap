// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Link components render correctly 1`] = `null`;
